src/components/Image/tests/Image.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Image from '..';
+import { mount } from 'enzyme';
+
+describe('source', () => {
```

Comment:
Add property/event for describe message. ` +describe('source property', () => { `